crates/ruff_cli/src/commands/format.rs

```diff
@@ -56,12 +57,9 @@ pub(crate) fn format(cli: &Arguments, overrides: &Overrides) -> anyhow::Result<E
             }
 
             let line_length = resolver.resolve(path, &pyproject_config).line_length;
-            // TODO(@konstin): Unify `LineWidth` and `LineLength`
-            let line_width = LineWidth::try_from(
-                u16::try_from(line_length.get()).expect("Line shouldn't be larger than 2**16"),
-            )
-            .expect("Configured line length is too large for the formatter");
-            let options = PyFormatOptions::from_extension(path).with_line_width(line_width);
+
+            let options = PyFormatOptions::from_extension(path)
+                .with_line_width(LineWidth::from(NonZeroU16::from(line_length)));
```

Comment:
It's unfortunate that `LineLength` cannot implement `Into<LineWidth>` because it doesn't depend on `ruff_formatter` but this isn't too bad. 

We probably want to have an independent `LineLength` inside of ruff too, so that settings no longer need to depend on `ruff`.